builtins/amp-img.js

```diff
@@ -63,6 +69,13 @@ export class AmpImg extends BaseElement {
     }
   }
 
+  /** @override */
+  onMeasureChanged() {
```

Comment:
What was the conclusion on the question of:
"If viewport size does NOT change, but `sizes` attribute of an `img` is updated, do any browser fetch and render a new high res image"?

If no, then we can just do this in viewport-size change event instead of measureChanged

Reply:
The answer to that question is yes for Firefox and Chrome, no for Safari (also sizes doesn't update on viewport change or image size change at all ever on Safari). The reason we have this in `onMeasureChanged` is because if the image changes size without a viewport change, updating the `srcset` will cause a refetch if the image size results in a different srcset being selected.